Clarified instructions around choosing K8S_VERSION

AlexChesser opened the pull request:
and added windows clarifications
Response to issue https://github.com/kubernetes/kubernetes/issues/17504

AlexChesser committed:
Clarified instructions around choosing K8S_VERSION
and added windows clarifications

derekwaynecarr commented on docs/getting-started-guides/docker.md:
````diff
@@ -226,6 +229,29 @@ the cluster, you need to first kill the kubelet container, and then any other co
+    curl https://gcr.io/v2/google_containers/hyperkube/tags/list -H "Authorization: Bearer AOR7SmExIINGsbUORS2fuqpY2O7r3JPiLJqX54l4eBqbCIivHYEbXN08/NB/BEtMHNuA9ZI5j7Zgg+9ES+H1b1/RsHnrt8YmV8Fo26jTEB+aDHQmMMjciKuKkL/Y8YdImhba9RKMKVtAyMlFJLweYcUN6fq1PLaejX4x760fCNGW2nlfGnNqnj851alZIEmW8pIIknr4n7ConRluNQWeXlOw7DyW132Nmf+UYtgWP3aYu9Y2wRqoB40t6r7ElMQZXjIbz/uawQc4k+3WIVsYYyYS7ax5ofGfudnfdfRxGLfh8fo8LbQl ...
+   ```   
+   
+4. The resulting array will have a list of current valit entries for ${K8S_VERSION} as of this writing v1.1.1 is the latest available result:   
````
typo: current valid entries

derekwaynecarr commented:
Rebase and fix typo needed.  Thanks for the PR

AlexChesser commented:
Hi @derekwaynecarr - I've fixed the typo and rebased the correction into a single commit. Hopefully that's what you mean by "rebase" I've also noticed that my current master is some ~800 commits behind kubernetes - master.  

If you mean that I need to merge that before you can accept my PR I would *love* a quick tip on how to pull from the origin/master repo I that I forked off of. (though I could always check stackoverflow too, but if you know off the top of your head that might be faster... if it matters)

derekwaynecarr commented:
@AlexChesser - sorry, needs another rebase something else got in the queue first.

derekwaynecarr commented:
Travis failure looks like you may need to run `hack/update-generated-docs.sh` after your next rebase.

Thanks!

fejta commented:
This PR has had no meaningful activity for multiple months. If it is still valid please rebase, push a new commit and reopen the PR. Thanks!
